airbyte-integrations/connectors/destination-clickhouse-v2/src/main/kotlin/io/airbyte/integrations/destination/clickhouse_v2/client/ClickhouseAirbyteClient.kt

```diff
@@ -101,8 +106,77 @@ class ClickhouseAirbyteClient(
         tableName: TableName,
         columnNameMapping: ColumnNameMapping
     ) {
-        // TODO: ("Not yet implemented")
-        // ensure transitioning from dedupe to non-dedupe and back works
+        val tableSchema = client.getTableSchema(tableName.name, tableName.namespace)
```

Comment:
I found I had to import `ClickhouseFinalTableNameGenerator` in the direct loader to get the proper name / namespace (e.g. namespace kept being null, etc.) for fetching the schema 🤔